boards/arm/reel_board/doc/index.rst

```diff
@@ -414,6 +428,15 @@ Then build and flash the application in the usual way.
    :board: reel_board
    :goals: build flash
 
+.. note::
+   Please use reel_board_v2 to build a application for the board equipped with
+   the new display E0213A22.
```

Comment:
Add a link to the section above that shows how to figure out which reel board you have:
```
the `E0213A22 display <reel_display_v2>`_.
```
But the same question here about the E0213A22 vs. GDEH0213B72 display ID.